P5/Source/Specs/charDecl.xml

```diff
@@ -52,4 +52,4 @@ $Id$
   <listRef>
     <ptr target="#D25-20"/>
   </listRef>
-</elementSpec>
+</elementSpec>
```

Comment:
I wonder if it came in when I was rebasing this branch...I'll see about reverting it.

Reply:
And I'll see about altering the examples--I was worried that the issue was the use of keel, chalder, and bushel twice in the *same* example. (I like that example a lot because it shows why we'd want to use `unitDecl` in the first place: it shows differences in these units of measure over time periods.) I'll try to come up with something different for the others so we have some variety.

Reply:
Naoki has some cool Japanese examples from his original work. Wouldn't it be nice to include them?

Reply:
Yes—the code needs to be updated, though, and they don’t demonstrate the use of `@formula`. I was thinking about those too. The example I am featuring seems to be one of his too, though using English units.

Reply:
Reverted charDecl.xml to its former state. Now looking into the examples...